lib/dor/update_marc_record_service.rb

```diff
@@ -103,12 +103,13 @@ def file_id
         if @druid_obj.datastreams['contentMetadata'].ng_xml
           resources = @druid_obj.datastreams['contentMetadata'].ng_xml.xpath('//contentMetadata/resource')
           resources.detect do |res|
-            children = res.children if res.attr('type') == 'image' || res.attr('type') == 'page'
+            # Added thumb based upon recommendation from Lynn McRae for future use
+            children = res.children if res.attr('type') == 'image' || res.attr('type') == 'page' || res.attr('type') == 'thumb'
```

Comment:
This may be more stylistic than anything, but this could possibly be more succinctly written as `if %w(image page).include?(res.attr('type'))`.